# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../csprng/0004-utils-fix-csprngd-to-include-limits.h.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From 169ca463e9819fa06d70730eb95e440bb1f6a2a0 Mon Sep 17 00:00:00 2001
From: Christian Wiese <chris@opensde.org>
Date: Thu, 5 Sep 2013 02:08:09 +0200
Subject: [PATCH] utils: fix csprngd to include limits.h

---
 utils/csprngd.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/utils/csprngd.c b/utils/csprngd.c
index 31cf51e..4115566 100644
--- a/utils/csprngd.c
+++ b/utils/csprngd.c
@@ -62,6 +62,7 @@ mpstat -P ALL 2
 #define _GNU_SOURCE
 #include <stdlib.h>
 #include <stdio.h>
+#include <limits.h> /* for INT_MAX, LONG_MAX */
 
 #include <csprng/csprng.h>
 #include <csprng/helper_utils.h>
-- 
1.7.2.3

